src/System.Net.Security/tests/FunctionalTests/SslStreamSystemDefaultsTest.cs

```diff
@@ -95,6 +91,32 @@ public TestBase()
                 }
             }
 
+            private bool ServerCertCallback(object sender, X509Certificate certificate, X509Chain chain, SslPolicyErrors sslPolicyErrors)
+            {
+                switch (sslPolicyErrors)
+                {
+                    case SslPolicyErrors.None:
+                    case SslPolicyErrors.RemoteCertificateChainErrors:
+                    case SslPolicyErrors.RemoteCertificateNameMismatch:
+                        return true;
+                    case SslPolicyErrors.RemoteCertificateNotAvailable:
+                        if (PlatformDetection.IsWindows7 && !Capability.IsTrustedRootCertificateInstalled())
```

Comment:
>If the problem is that the root cert isn't installed, that feels like it should be a chain error, not certificate not available...

That's what I though too., but then I started looking at the other tests and found [this code used by the `CertificateValidationClientServer_EndToEnd_Ok` test that does this.](https://github.com/dotnet/corefx/blob/d9c476a19fc081aefc7e5f758cf5b6a48c9a37e9/src/System.Net.Security/tests/FunctionalTests/CertificateValidationClientServer.cs#L166) That test is from where I copied the comment below. 

I'm completely new to this space, so it's certainly possible that I equated the two scenarios as being equivalent when actually there is something else at play in `CertificateValidationClientServer_EndToEnd_Ok` that requires that behavior, but if so then I was unaware of it.